engine/pom.xml

```diff
@@ -802,6 +812,16 @@
           <groupId>org.camunda.connect</groupId>
           <artifactId>camunda-connect-connectors-all</artifactId>
           <version>${project.version}</version>
```

Comment:
📓 The usage of the upstream library here is for engine plugins via `check-plugins` profile. We no longer have CI for this profile so I ran the tests locally with it and they passed successfully.

Reply:
Thanks!